### ISSUE TITLE
[docs] Highlight active section in right navbar

### ISSUE THREAD

**Issue by dholbach** [fluxcd/website#118]
<img width="344" alt="Screenshot 2021-03-24 at 15 05 58" src="https://user-images.githubusercontent.com/1346979/112316421-63682d00-8cab-11eb-9564-7ffe49a271d5.png">

docsy vs. mkdocs

<img width="350" alt="Screenshot 2021-03-24 at 15 07 41" src="https://user-images.githubusercontent.com/1346979/112316503-7418a300-8cab-11eb-80d1-49562c24632a.png">

**Comment by kingdonb** [comment]
@hiddeco Were you saying that you have a piece of code which basically does this already somewhere, or is my memory failing me?

**Comment by hiddeco** [comment]
There is one in the previous revision of the website, before moving to the Docsy theme.

**Comment by dholbach** [comment]
Wasn't that in `mkdocs`?

**Comment by hiddeco** [comment]
No, I also implemented it in the rework of the site I made myself I think.

**Comment by dholbach** [comment]
ok, cool

**Comment by juozasg** [comment]
@hiddeco can you help me with finding or describing that code? I would like to close this ticket. It should not take long to write that code using IntersectionObserver WebAPI or a height calculation method like this: https://codepen.io/zbymhz/pen/ExjRNwZ

**Comment by hiddeco** [comment]
Can't seem to easily find it, but from what I remember it noticed IDs entering and leaving the viewport using an IntersectionObserver or something something.

**Comment by dholbach** [comment]
The OpenFaas docs use `mkdocs` -it works there: https://docs.openfaas.com/community/
